apps/address-book/app/index.html

```diff
@@ -7,7 +7,7 @@
 
 <body>
 
-    <div id="root"></div>
+    <div id="addressbook"></div>
```

Comment:
Do we need to rename this?

Reply:
We don't need to rename it, but we had a mix of id=root and id=something in applications, so now we only have id=somethings. We could as well have id=roots :) What's the pro/con argument?

Reply:
I like consistency! And this is a small change, so it may as well piggy-back on this one. I wasn't sure if the updated packages conflicted with the `id="root"`

Reply:
This should be just a naming opinion, my point about that is: I also like consistency, and semantically it sounds better for me to use the name of the app instead of the "cold" `root`

But I guess is really unimportant from the code / functionality perspective in any case